blob/blob.go

```diff
@@ -505,6 +503,15 @@ type SignedURLOptions struct {
 	Expiry time.Duration
 }
 
+// ReaderOptions controls Reader behaviors.
+type ReaderOptions struct {
+	// ReadLength sets the total number of bytes to read. 0 is interpreted as
+	// no limit. Must be >= 0.
+	ReadLength int64
```

Comment:
nit: why not just Length? It's not like anyone is going to be confused by what that means, since it's in a ReaderOptions struct and you document it. Also, you don't say ReadOffset.

Reply:
Done.